### ISSUE TITLE
[UX] [multilingual] Translated blocks are too hard to use

### ISSUE THREAD

**Issue by sthomen** [backdrop/backdrop-issues#3514]
My non-technical co-worker seems to find translated blocks difficult to work with, he would continually follow the "Configure block" contextual link to translate it, but then end up editing the source translation.

I would suggest a contextual link to translate the content and/or that the block configuration popup be more obvious about the content being the source translation (although my co-worker probably wouldn't have much of an idea what source translation means). Perhaps disabling (or with an option to disable) editing the source content directly from the configuration popup would produce fewer mistakes.

Discuss. :-)

As a stop-gap I wrote a [small module](https://github.com/sthomen/block_translation_link) that adds the contextual link.

**Comment by laryn** [comment]
I agree they are difficult. I [just tried to figure them out](https://forum.backdropcms.org/forum/multilingual-blocks#comment-991) and while I was glad to see that the capability exists, I think there's a lot of room to improve. I created a custom reusable block and placed it on a layout, but then couldn't figure how to translate it. When I finally decided to try the **Structure > Custom Blocks** section, it seemed to require that I create a new copy of the block and translate that new copy. Then I had to troubleshoot for a while before I figured out that it was the old non-multilingual block that was on the Layout, not the new duplicate it made me create.

**Suggestions/Questions:**

 - When creating a new custom block on a multilingual site, the "reusable" text already indicates that it has to be reusable in order to be translated. 
![Screen Shot 2019-05-02 at 2 29 07 PM](https://user-images.githubusercontent.com/871421/57101352-aa7b5c80-6ce6-11e9-9d39-f0dae94402f1.jpg)
Can we expose the language dropdown right here when "reusable" is selected?
 - Can we add the contextual link that @sthomen proposes?
 - Can we adjust the Configure link depending on the currently viewed version of the block? (e.g. if you are viewing the English translation you go to edit the English version; if you are viewing the French translation and click "Configure", you edit the French translation?)
